chromadb/cli/cli.py

```diff
@@ -97,5 +100,23 @@ def docs() -> None:
     webbrowser.open("https://docs.trychroma.com")
 
 
+@app.command()  # type: ignore
+def migrate(
```

Comment:
I think its a bit odd that you have to specify the tenant and database, and would prefer if we added some tooling to migrate all of them. This feels like pushing our constraints (no list tenants) onto our users and delivering a worse UX.

Reply:
My thought here was that if you are using the tenant and database args, you know what you're doing and what you want to migrate. I could instead crawl the entire sysDB too. Let's discuss. 

(the error here is I forget to pass these args, fixed.)